ABOUT.md

```diff
@@ -1,9 +1,106 @@
+Anyone - scientists, data enthusiasts, non-profit organisations, hackers, citizens, journalists, students - should be able to use ML tools for free.
+
+The Shogun ML toolbox promotes this mission through its features (*accessible*, *open source* and *good*) and a focus on ML education and developing its contributors.
+
+We are proud to be a big and very active open source ML platform.
+You can find the statistics of Shogun's development activity on [ohloh](https://www.openhub.net/p/shogun).
+
+## <a name="features"></a> Shogun features - *accessible*, *open source* and *good*
+
+### Accessible
+
+ * Shogun is **not limited to a single language**: 
+ you can use the toolbox through a unified interface from C++, Python, Octave, R, Java, Lua, C#, Ruby, etc. See [examples](http://shogun.ml/cookbook/latest/). 
+ Thus Shogun is independent of trends in computing languages and importantly, this allows you to use Shogun as a vehicle to expose your algorithm to multiple communities.  
+ * We use [SWIG](http://www.swig.org/) to enable *bidirectional* communication between C++ and all target languages.
+ * Shogun **runs natively under Linux/Unix, MacOS, and Windows**.
+ * Anyone can **try Shogun with minimal effort, for free**: 
+ simply by connecting to a [jupyter](http://jupyter.org/) notebook, users can run a docker instance on the Shogun [cloud](https://cloud.shogun.ml/hub/home)
+ (We offer this service for free through external donations).
+ * We offer as many binary installation packages as possible.
+ You can also just download our Docker image and run the Shogun cloud locally.
+ * Shogun does not re-invent the wheel, but offers **bindings to other sophisticated libraries including**: 
+ [LibSVM](http://www.csie.ntu.edu.tw/~cjlin/libsvm/)/[LibLinear](http://www.csie.ntu.edu.tw/~cjlin/liblinear/), 
+ [SVMLight](http://svmlight.joachims.org/), 
+ [LibOCAS](http://cmp.felk.cvut.cz/~xfrancv/ocas/html/), 
+ [libqp](http://cmp.felk.cvut.cz/~xfrancv/libqp/html/), 
+ [VowpalWabbit](http://www.hunch.net/~vw/), 
+ [Tapkee](http://tapkee.lisitsyn.me/), 
+ [SLEP](http://www.public.asu.edu/~jye02/Software/SLEP/), 
+ [GPML](http://www.gaussianprocess.org/gpml/code/matlab/doc/) and more. 
+ Future plans include to offer bindings to popular packets such as tensorflow and stan.
+
+### Free
+
+ * Shogun is **community based** and **non-commercial**.
+ * Licence: Shogun is currently released under GPLv3. 
+ We are working towards a BSD compatible repository and all newly added code is required to be BSD3 compliant.
+ This means companies can use the code without having to turn their code-base to GPL.
+
+### Good (is what we aim for)
+
+ * Shogun provides **efficient** implementation of all standard ML algorithms.
+  * Our implementation is [**competitive**](https://github.com/zoq/benchmarks) and we contribute to the benchmarking project by MLPack and Markus Edel (a framework to compare algorithms from different ML libraries).
```

Comment:
You mix "ours" and "Shogun's". Happens at a couple of places in the text. Should be addressed globally (I wont comment everywhere)